installer/targets/openface/dlib.bash

```diff
@@ -29,6 +29,9 @@ sudo sh -c 'echo "/usr/lib/openblas-base/" > /etc/ld.so.conf.d/libopenblas-base.
 #Finally, run ldconfig to update the cache.
 sudo ldconfig
 
+#Remove source files
+sudo rm -rf ~/src
```

Comment:
I put a lot of my own programs in `~/src` so this is obviously wrong.

Reply:
Why not dowload the stuff in /tmp? The binaries are copied and the other files aren't used anymore, right?